airflow/providers/apache/kafka/shared_utils.py

```diff
@@ -0,0 +1,53 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+import importlib
+from typing import Any, Callable
+
+
+def _get_callable(function_string: str) -> Callable:
```

Comment:
There is a [native `import_string` util function](https://github.com/apache/airflow/blob/4f3751aab677904f043d3c0657eb8283d93a9bbd/airflow/utils/module_loading.py#L26-L41) that might be used instead?

Reply:
there is. 

The doc string indicates "attr/class", while true it is pretty much the same code - i'm not sure if the intent will stay consistent in the future.